internal/controller/controllers/agent_controller_test.go

```diff
@@ -578,6 +578,33 @@ var _ = Describe("agent reconcile", func() {
 		Expect(conditionsv1.FindStatusCondition(agent.Status.Conditions, v1beta1.SpecSyncedCondition).Status).To(Equal(corev1.ConditionTrue))
 	})
 
+	It("Agent update status with no cluster id ", func() {
```

Comment:
it doesn't look like the coverage have changed. 
I suggest to check a specific flow and fail `UnbindHostInternal` similar to https://github.com/openshift/assisted-service/blob/67e0bc1275f07acf46a2f4531f6984e6f7edb7f3/internal/controller/controllers/agent_controller_test.go#L542